tests/test_priority_pool.py

```diff
@@ -31,27 +50,13 @@ def get_pools(self):
         PrioritizedTaskPool(dummy_pool_func, name="B", max_batch_size=1),
     )
 
+    # Simulate requests coming from ConnectionHandlers
+    proc = mp.context.ForkProcess(target=_submit_tasks, args=(pools, results_valid))
+    proc.start()
```

Comment:
Moving that before `Runtime.start()` since **(a)** that's how real server works and **(b)** so we don't fork after creating active threads (this provably improves test stability)